src/config/dynamic-theme-fixes.config

```
@@ -6611,6 +6611,13 @@ CSS
 
 ================================
 
+estadao.com.br
+
+INVERT
+#puzzleHolder 
```

Please add the following suggestion, so the letters won't turn dark.
```suggestion
#puzzleHolder
.puzzleField
```

---

Thanks!